examples/misc.ipynb

```diff
@@ -674,7 +674,7 @@
      "name": "stderr",
      "output_type": "stream",
      "text": [
-      "becquerel/core/spectrum.py:556: SpectrumWarning: Subtraction of counts-based specta, spectra have been converted to CPS\n",
+      "/Users/jvavrek/becquerel/becquerel/core/spectrum.py:713: SpectrumWarning: Subtraction of counts-based specta, spectra have been converted to CPS\n",
```

Comment:
Remove your username here

Reply:
That's just the output from a warning in the notebook